proto/engine/v1/json_marshal_unmarshal.go

```diff
@@ -74,6 +75,11 @@ func (e *ExecutionBlock) UnmarshalJSON(enc []byte) error {
 	// them into a list of geth transaction objects.
 	txs := make([]*gethtypes.Transaction, len(txsList))
 	for i, tx := range txsList {
+		// If the transaction is just a hex string, do not attempt to
+		// unmarshal into a full transaction object.
+		if txItem, ok := tx.(string); ok && strings.Contains(txItem, "0x") {
```

Comment:
wonder if we should use strings.Prefix instead of Contains

Reply:
Great feedback!